code-challenges.test.js

```diff
@@ -43,12 +105,41 @@ var arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
 
 // b) Create the function that makes the test pass.
 
-
+// First I need to create a function called includeVowel
+// It takes the argument of an array
+// I am going to iterate over the array and for each value at the index I want to check if it includes the letter a
+// If it does that value will get pushed into an empty array called finalArray
+// finalArray will get returned
+
+
+const includeVowel = (array) => {
+   lowerArray = array.map(value =>{
+    return value.toLowerCase()
+  })
+  var arrayA = []
+lowerArray.map(value =>{
+  if(value.includes("a")){
+    arrayA.push(value)
+}})
+  return arrayA.map(value => value.charAt(0).toUpperCase() + value.slice(1))
+    }
+// So I initially did lines 119-122 basically as my first attempt but I used filter and it worked except for the capital A. I tried to think of/ook up a regex way to account for lowercase and capital letters but it didn't work. So I decided to lower case the whole array, push the a words to an empty array and then put it back to uppercase. I feel like there is an easier way to do this, but this path works as well.
```

Comment:
Correct, So you have the right idea with map
1.) we need iteration -> for, for each, map, filter
2.) we need a new array -> map, filter
3.) we need a smaller array than we started with (referred to as a subset) -> filter

filter uses a built in conditional and returns us a new array with only the values selected for.